src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -218,18 +218,22 @@ public bool TryWriteProperties(PropertiesWriter writer, IList<ProjectInfo> proje
         {
             projectData.Status = ProjectInfoValidity.InvalidGuid;
         }
+        else if (!File.Exists(projectData.Project.FullPath)) // If a project was created and destroyed during the build, it is no longer valid
```

Comment:
I would give an example. // When using ..., a project is temporarily generated during build and then removed.